externs/shaka/player.js

```diff
@@ -1294,6 +1295,13 @@ shaka.extern.DashManifestConfiguration;
  *   https://www.akamai.com/blog/performance/-using-ll-hls-with-byte-range-addressing-to-achieve-interoperabi
  *   <br>
  *   Defaults to <code>true</code>.
+ * @property {number} updatePeriod
+ *   Override the updatePeriod of the manifest. The value is in second
```

Comment:
My only complaint about this PR is the way this is phrased.  I'm going to make an edit to clean up the grammar.

Reply:
Sorry my French

Reply:
No worries!  English is stupid and hard sometimes.  For example, why is it "sometimes" and not "some times"?  I'm not sure.  That's just native speaker intuition for me.  And yet, other times, my native speaker intuition leads me to make stupid or lazy mistakes of my own.